src/twisted/protocols/newsfragments/8570.bugfix

```diff
@@ -0,0 +1,3 @@
+Doubled the MAX_LENGTH of twisted.protocols.basic.LineOnlyReceiver and
+twisted.protocols.basic.LineReceiver, from 16384 bytes to 32768 bytes, to
+support scenarios seen in the wild.
```

Comment:
I think that this needs to go to src/twistd/newsfragments

we don't have a separate section for protocol.

To comply with our standard

```suggestion
twisted.protocols.basic.LineOnlyReceiver and
twisted.protocols.basic.LineReceiver, can now handle lines up to 32768 bytes long. In previous versions, the limit was 16384 bytes.
```